src/include/duckdb/parser/query_node.hpp

```diff
@@ -23,6 +23,19 @@ enum QueryNodeType : uint8_t {
 	RECURSIVE_CTE_NODE = 4
 };
 
+struct CommonTableExpressionInfo;
+
+class CommonTableExpressionMap {
+public:
+	CommonTableExpressionMap();
+	CommonTableExpressionMap(const CommonTableExpressionMap &other);
+
+	unordered_map<string, unique_ptr<CommonTableExpressionInfo>> map;
+
+public:
+	string CTEToString() const;
```

Comment:
Can we call this `ToString()` for consistency?

Reply:
Sure, done